grafana/resource_builtin_role_assignment.go

```diff
@@ -226,7 +224,7 @@ func createOrRemove(meta interface{}, name string, changes []RoleChange) error {
 			err = client.DeleteBuiltInRoleAssignment(br)
 		}
 		if err != nil {
-			return errors.New(fmt.Sprintf("Error with %s %v", name, err))
+			return fmt.Errorf("error with %s %v", name, err)
```

Comment:
Nit: when wrapping an `error` use `%w` instead of `%v` so it works with `errors.Unwrap`
```suggestion
			return fmt.Errorf("error with %s %w", name, err)
```